packages/ui/src/components/va-input/components/VaTextarea/VaTextarea.vue

```diff
@@ -46,18 +48,22 @@ export default defineComponent({
     const textarea = ref<HTMLTextAreaElement | undefined>()
     const rowHeight = ref(-1)
     const height = ref(-1)
-    const { calculateRowHeight } = useTextareaRowHeight(textarea)
+    const { calculateRowHeight, calculateHeight } = useTextareaRowHeight(textarea)
 
-    const updateRowHeight = () => {
-      if (!props.autosize || !textarea.value) { return }
+    const isResizable = computed(() => {
+      return (props.autosize || props.maxRows || props.minRows !== 1) && textarea.value
+    })
 
-      rowHeight.value = calculateRowHeight()
+    const updateRowHeight = () => {
+      if (isResizable.value) {
+        rowHeight.value = calculateRowHeight()
+      }
     }
 
     const updateHeight = () => {
-      if (!props.autosize || !textarea.value) { return }
-
-      height.value = textarea.value?.scrollHeight || 0
+      if (isResizable.value) {
+        height.value = calculateHeight() || 0
```

Comment:
![UnTG7PDWeP](https://user-images.githubusercontent.com/23530004/146372372-68e17c6f-2d49-4f97-bc35-17fe99261990.gif)

Here is a bug in height calculation.

Reply:
fixed it